packages/snowpack/src/util.ts

```diff
@@ -177,22 +177,46 @@ export async function openInBrowser(
     : /brave/i.test(browser)
     ? appNames[process.platform]['brave']
     : browser;
-  if (process.platform === 'darwin' && /chrome|default/i.test(browser)) {
+  const isMac = process.platform === 'darwin' && /chrome|default/i.test(browser);
+  if (isMac) {
     // If we're on macOS, and we haven't requested a specific browser,
     // we can try opening Chrome with AppleScript. This lets us reuse an
     // existing tab when possible instead of creating a new one.
     try {
       await execa.command('ps cax | grep "Google Chrome"', {
         shell: true,
       });
-      await execa('osascript ../assets/openChrome.applescript "' + encodeURI(url) + '"', {
-        cwd: __dirname,
-        stdio: 'ignore',
-        shell: true,
-      });
+      let isChromeInitialized = false;
+      const openChrome = execa(
+        'osascript ../assets/openChrome.applescript "' + encodeURI(url) + '"',
+        {
+          cwd: __dirname,
+          stdio: 'ignore',
+          shell: true,
+        },
+      );
+      // Sometimes Chrome doesn’t respond; give it 3 seconds max
+      setTimeout(() => {
+        if (!isChromeInitialized) {
+          openChrome.cancel();
+          console.warn(
+            `Chrome not responding to Snowpack after 3s. Opening dev server in default tab.`,
+          );
+          open(url);
+        }
+      }, 3000);
+
+      try {
+        await openChrome;
+        isChromeInitialized = true;
```

Comment:
NIT: Would it be better to use `clearTimeout` instead of checking a flag?

Reply:
Yeah that’s a little cleaner you’re right 👍🏻